.github/workflows/appimage.yml

```diff
@@ -0,0 +1,57 @@
+name: Build AppImage rplidar module
+
+concurrency: 
+  group: ${{ github.workflow }}-${{ github.ref }}
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ 'main' ]
+    paths-ignore:
+      - 'README.md'
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  appimage:
+    name: AppImage Build Rplidar Module
+    strategy:
+      matrix:
+        include:
+          - arch: [x64, qemu-host]
+            image: ghcr.io/viamrobotics/canon:amd64-cache
+            platform: linux/amd64
+          - arch: [arm64, qemu-host]
+            image: ghcr.io/viamrobotics/canon:arm64-cache
+            platform: linux/arm64
+    runs-on: ${{ matrix.arch }}
+    container:
+      image: ${{ matrix.image }}
+      options: --platform ${{ matrix.platform }}
+    timeout-minutes: 30
+    outputs:
+      date: ${{ steps.build_test_app.outputs.date }}
+
+    steps:
+      - name: Check out main branch code
+        uses: actions/checkout@v3
+      
+      - name: Build appimage
+        run: |
+          make install-swig
+          make appimage
```

Comment:
If you continue to use the BUILD_CHANNEL variable (which I think you can remove, if you want to), you will need to set it to "latest" here.